doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -322,27 +322,35 @@ components and libraries. Please consult the instructions in the
 Installing R
 ============
 
-Can be installed with :fixedwidthplain:`yum`::
+For RHEL/CentOS, the EPEL distribution is strongly recommended::
```

Comment:
```suggestion
For RHEL/CentOS, the EPEL distribution is strongly recommended:
```

To avoid this error:

/Users/pdurbin/github/iqss/guides.dataverse.org/doc/sphinx-guides/source/installation/prerequisites.rst:331: WARNING: Literal block expected; none found.

It's the double :: that's causing trouble. Removing one fixes it. You could change the remaining colon to a period. Probably looks better.